### PR TITLE
Add linting

### PR DESCRIPTION
**Pull request by tijmenb**
🚔  Woop wop it's the sound of Rubocop 👮 

https://trello.com/c/6QSFbxIv

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+inherit_gem:
+  govuk-lint: "configs/rubocop/all.yml"
```
Comment (erino) [inline .rubocop.yml]: Should all of the Rails specific cops be disabled?

Comment (tijmenb) [reply .rubocop.yml]: I'm not sure if they run / interfere with anything?

Comment (erino) [reply .rubocop.yml]: They're not failing anything at the moment, if it's a problem it can always be disabled later

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+AllCops:
+  TargetRubyVersion: 2.6
```
Comment (erino) [inline .rubocop.yml]: I don't think we need this [given that Rubocop](https://github.com/rubocop-hq/rubocop/blob/master/manual/configuration.md#setting-the-target-ruby-version) uses [.ruby-version](https://github.com/alphagov/govuk-docker/blob/master/.ruby-version)

Comment (tijmenb) [reply .rubocop.yml]: https://travis-ci.org/alphagov/govuk-docker/builds/553761983 🤷‍♂

Comment (erino) [reply .rubocop.yml]: 🤷‍♂

Comment (erino) [reply .rubocop.yml]: I've found the problem, the gems are vendored and Rubocop is checking all of them

vendoring: https://travis-ci.org/alphagov/govuk-docker/builds/553761983#L565
all: https://travis-ci.org/alphagov/govuk-docker/builds/553761983#L650

File: .rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+inherit_gem:
+  govuk-lint: "configs/rubocop/all.yml"
+
+AllCops:
```
Comment (erino) [inline .rubocop.yml]: Can we excluded vendored ruby
```yaml
Exclude:
    - 'vendor/**/*'
```

**Pull request merged**